frontend/packages/dev-console/integration-tests/features/guided-tour/quick-starts/quick-starts-cluster-overview.feature

```diff
@@ -11,50 +11,49 @@ Feature: Add Quick Starts card on cluster overview
 
 
         @regression @to-do
-        Scenario: Quick Starts card on Cluster Overview: QS-04-TC01
+        Scenario: Build your guided documentation card on Cluster Overview: QS-04-TC01
             Given user is in administrator perspective
```

Comment:
```suggestion
            Given user is at administrator perspective
```

Reply:
Also, you have already added it in the background. Is it necessary to add it here?

Reply:
Updated